docs/UserGuide.md

```diff
@@ -256,14 +284,22 @@ Example(s) of usage:
 * `list` followed by `delete 2` deletes the 2nd item stored in the recipe book.
 *  `find chicken` followed by `delete 1` will delete the 1st recipe in the displayed results of the find command.
 
-> Deletes the dish at the specified `INDEX` of the latest list that was displayed.
->
-> The index **must be a positive integer** 1, 2, 3, …
+<div markdown="block" class="alert alert-info">
+
+* Deletes the dish at the specified `INDEX` of the latest list that was displayed.
```

Comment:
`* Deletes the dish at the specified `INDEX` of the current list.` 
Using past tense`latest list that was displayed` could cause confusion on whether its the previous one or the current one.